install/installer/traf_getHadoopNodes

```diff
@@ -31,20 +29,22 @@ echo "***INFO: Getting list of all $HADOOP_TYPE nodes"
 
 curlRC=0
 if [[ $HADOOP_TYPE == "cloudera" ]]; then
-   curl -su $ADMIN:$PASSWORD http://$URL/api/v6/cm/deployment > tempFile
+   curl -k -su $ADMIN:$PASSWORD $URL/api/v10/hosts > tempFile
    curlRC=$?
```

Comment:
It is being used a few lines down. 

if [ $curlRC != 0 ]; then
   echo "***ERROR: Unable to get list of hosts from $HADOOP_TYPE (RC=$curlRC)"
   echo "***ERROR: curl command failed."
   exit -1
fi